sdk/tables/azure-data-tables/tests/test_table_encryption.py

```diff
@@ -52,10 +52,10 @@
 # #     TablePayloadFormat,
 # #     TablePermissions,
 # # )
-# from azure.data.tables._shared._error import (
+# from azure.data.tables._error import (
```

Comment:
why is this whole file commented out

Reply:
I believe it's because we haven't looked into encryption for tables yet. There's an open issue: https://github.com/Azure/azure-sdk-for-python/issues/12310